atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -959,6 +959,17 @@ private int estimateSize(List<RowResult<Value>> rangeRows) {
             }
         }
 
+        if (AtlasDbConstants.hiddenTables.contains(tableRef)) {
```

Comment:
I'm aware of that, but I thought it wouldn't hurt to add another check here in case something didn't call it.  It makes sense to explicitly assert the condition here too because this is actually where we are changing behavior compared to a normal snapshot transaction.